<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YuvRgbPlayerClass</class>
 <widget class="QWidget" name="YuvRgbPlayerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YuvRgbPlayer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_video">
        <property name="text">
         <string>Video1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_video2">
        <property name="text">
         <string>Video2</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_video_set" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <item row="5" column="0">
          <widget class="QLabel" name="label_fps">
           <property name="text">
            <string>帧率</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QComboBox" name="comboBox_fmt">
           <item>
            <property name="text">
             <string>YUV420P</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>RGBA</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ARGB</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>BGRA</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>RGB</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QPushButton" name="pushButton_open">
           <property name="text">
            <string>打开</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_height">
           <property name="text">
            <string>高度</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="lineEdit_fps"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_width">
           <property name="text">
            <string>宽度</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_width"/>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_height"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget_video2_set" native="true">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QComboBox" name="comboBox_fmt2">
           <item>
            <property name="text">
             <string>YUV420P</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>RGBA</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ARGB</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>BGRA</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>RGB</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="pushButton_open2">
           <property name="text">
            <string>打开</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_width2"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_width2">
           <property name="text">
            <string>帧率</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_height2"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_height2">
           <property name="text">
            <string>高度</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_width2_2">
           <property name="text">
            <string>宽度</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_fps2"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="yuvrgbplayer.qrc"/>
 </resources>
 <connections/>
</ui>
